packages/web-react/src/components/Tooltip/stories/Tooltip.stories.tsx

```diff
@@ -1,10 +1,26 @@
+import React, { useEffect, useRef, useState } from 'react';
 import { Markdown } from '@storybook/blocks';
 import type { Meta, StoryObj } from '@storybook/react';
-import React from 'react';
-import { Tooltip, TooltipWrapper } from '..';
+
+import { SpiritTooltipProps } from '../../../types';
 import { Button } from '../..';
-import { Placements } from '../../../constants';
 import ReadMe from '../README.md';
+import { Tooltip, TooltipPopover, TooltipTrigger } from '..';
+
+const Placements = [
```

Comment:
We have some placement constants in `constants/dictionaries`, could we please use the dictionary here?

Reply:
yes, Placements changed to point to this dictionary